blackbelt/handle_github.py

```diff
@@ -409,3 +410,31 @@ def create_release(ref, payload, description, repo_info):
     if r.status_code != 201:
         print(r.json())
         raise ValueError("Create github release ended with status code %s: %s" % (r.status_code, r))
+
+def run_grunt_in_parallel(grunt_commands):
+    """ Helper function to run grunt commands (e.g. deploy or slug manipulation) for multiple environments in parallel"""
+
+    commands = []
+    for command in grunt_commands:
+        with tempfile.NamedTemporaryFile(delete=False) as f:
```

Comment:
Why this over `mkstemp`?

Reply:
Because I wasn't aware about it ;-) Since I don't need `delete` attribute for NamedTemporaryFile, I can switch.

Reply:
Well... I'll probably stay with `NamedTemporaryFile`; `mkstemp` doesn't support context manager (a.k.a. `with ...`), therefore - AFAIK - I'd need to take care of closing the handler, which would complicate whole flow.